bindings/python/tests/fdbcli_tests.py

```diff
@@ -442,24 +463,33 @@ def throttle(logger):
     assert enable_flag == "`0'"
     # TODO : test manual throttling, not easy to do now
 
+
 def wait_for_database_available(logger):
     # sometimes the change takes some time to have effect and the database can be unavailable at that time
     # this is to wait until the database is available again
     while not get_value_from_status_json(True, 'client', 'database_status', 'available'):
         logger.debug("Database unavailable for now, wait for one second")
         time.sleep(1)
 
+
 if __name__ == '__main__':
-    # fdbcli_tests.py <path_to_fdbcli_binary> <path_to_fdb_cluster_file> <process_number>
-    assert len(sys.argv) == 4, "Please pass arguments: <path_to_fdbcli_binary> <path_to_fdb_cluster_file> <process_number>"
+    # fdbcli_tests.py <path_to_fdbcli_binary> <path_to_fdb_cluster_file> <process_number> [external_client_library_path]
+    assert len(sys.argv) == 4 or len(
+        sys.argv) == 5, "Please pass arguments: <path_to_fdbcli_binary> <path_to_fdb_cluster_file> <process_number> [external_client_library_path]"
+    # set external client library
+    if len(sys.argv) == 5:
+        external_client_library_path = sys.argv[4]
+        # disable local client and use the external client library
+        os.environ['FDB_NETWORK_OPTION_DISABLE_LOCAL_CLIENT'] = ''
+        os.environ['FDB_NETWORK_OPTION_EXTERNAL_CLIENT_LIBRARY'] = external_client_library_path
```

Comment:
Use `fdb.options` to set network options instead of environment variables

Reply:
Changed to add env variables in _Popen_